packages/x-components/src/design-system/components/input-group/default.scss

```
@@ -2,10 +2,12 @@
 
```

In `input/default.scss` there are some styles that are being applied to a placeholder:

```suggestion
  &::placeholder {
    // typography
    color: var(--x-color-text-input-placeholder-default);
    font-family: var(--x-font-family-input-placeholder-default);
    font-size: var(--x-size-font-input-placeholder-default);
    font-weight: var(--x-number-font-weight-input-placeholder-default);
    line-height: var(--x-size-line-height-input-placeholder-default);
  }
```

I think that you should do the same for `&-placeholder` like you are doing in this file